qt.BUILD

```diff
@@ -114,4 +95,18 @@ cc_library(
         "-lQt53DQuickScene2D",
         "-lQt53DRender",
     ],
+    # Available from Bazel 4.0.0
+    # target_compatible_with = ["@platforms//os:linux"],
+)
+
+filegroup(
+    name = "uic",
+    srcs = ["bin/uic.exe"],
```

Comment:
I was not aware that filegroup would work with absolute paths that are not relative to the WORKSPACE file, I will try it out, it would be a much more elegant solution.

Reply:
I tried but I'm getting this error, any ideas?
`ERROR: /home/limdor/.cache/bazel/_bazel_limdor/82e4b01d7dfd8c03a2e6b25ed0350e44/external/qt/BUILD.bazel:144:10: @qt//:uic: invalid label '/usr/bin/uic' in element 0 of each branch in select expression of attribute 'srcs' in 'filegroup' rule (including '@platforms//os:linux'): invalid target name '/usr/bin/uic': target names may not start with '/'`
This is what I tried:
```
filegroup(
    name = "uic",
    srcs = select({
        "@platforms//os:linux": ["/usr/bin/uic"],
        "@platforms//os:windows": ["bin/uic.exe"],
    }),
    visibility = ["//visibility:public"],
)
```

Reply:
Ahh you're right, I forgot that you can't do system-local paths like that. It might be possible to use something like `qt_configure.bzl` to determine the path to the binaries and use `local_repository()` to add targets for them.

This should be fine for now, but longer-term, I'd like to unify this between linux and windows so that there's a single :uic target somewhere that can be used in other rules without having to worry about platform.

Reply:
I would expect that with toolchains this goes away and gets unified. I would expect that is possible to do something like tool_path and then this would be part of the toolchain and accessible and callable in the sam way for linux and windows.